libraries/AP_RSSI/AP_RSSI.cpp

```diff
@@ -128,6 +128,13 @@ float AP_RSSI::read_receiver_rssi()
         case RssiType::RSSI_RC_CHANNEL_VALUE :
             receiver_rssi = read_channel_rssi();
             break;
+        case RssiType::RSSI_RECEIVER : {
+            int16_t rssi = hal.rcin->get_rssi();
+            if (rssi != -1) {
+                receiver_rssi = rssi / 255.0;
+            }
```

Comment:
I compiled this up and tested it, walking around the block with my transmitter chattering at me - it works as well as it ever did, good deal.

I've also put together a branch with the additional changes to the ranges I'd like to see:

https://github.com/StewLG/ardupilot/tree/pr-receiver-rssi-slg

I know it can be hard to feel another man's pain until you've spent some time walking in his use cases, but I think this is beneficial to the feature. That said, it would be better to have it without ranges than not have it at all, so I'm keen to see this feature merged either way.

Thanks again.